app/index.js

```diff
@@ -1025,11 +1025,9 @@ module.exports = JhipsterGenerator.extend({
             if (this.useSass) {
                 this.template(WEBAPP_DIR + 'scss/main.scss', WEBAPP_DIR + 'scss/main.scss');
                 this.template(WEBAPP_DIR + 'scss/vendor.scss', WEBAPP_DIR + 'scss/vendor.scss');
+            } else {
+                this.template(WEBAPP_DIR + 'content/css/main.css', WEBAPP_DIR + 'content/css/main.css');
             }
-            // this css file will be overwritten by the sass generated css if sass is enabled
```

Comment:
we also had similar code before but there were lot of tickets regarding run failure and we wanted something to be there to startup even if build fails for any reason. I have had weird issue of build not running after project generation many times. SO its better to have this by default can you plz put it back